examples/rapids-blog/02-rapids-train.ipynb

```diff
@@ -0,0 +1,183 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "059ae374-4dbe-47fb-a881-96fc2295ee61",
+   "metadata": {},
+   "source": [
```

Comment:
This cell should start with the text:

```
This notebook gives an example of creating a regression model to predict the price of train tickets using CuDF dataframe and CuML. Hence instead of utilizing CPUs our computations will run on GPUs.
```